packages/tatum-core/src/model/request/RegisterErc20Token.ts

```diff
@@ -0,0 +1,68 @@
+import { Type } from 'class-transformer'
+import {
+  IsIn,
+  IsNotEmpty,
+  IsNumber,
+  IsNumberString,
+  IsOptional,
+  Length,
+  Matches,
+  Max,
+  MaxLength,
+  Min,
+  ValidateNested,
+} from 'class-validator'
+import { Fiat } from '../response'
+import { Currency } from './Currency'
+import { CustomerUpdate } from './CustomerUpdate'
+
+export class RegisterErc20Token {
+  @IsNotEmpty()
+  @Length(1, 30)
+  @Matches(/^[a-zA-Z0-9_]+$/)
+  public symbol: string
+
+  @IsNotEmpty()
+  @IsNumberString()
+  @MaxLength(38)
+  @Matches(/^[+]?((\d+(\.\d*)?)|(\.\d+))$/)
+  public supply: string
+
+  @IsNotEmpty()
+  @IsNumber()
+  @Min(0)
+  public decimals: number
+
+  @IsNotEmpty()
+  @Length(1, 100)
+  public description: string
+
+  @Length(3, 50)
+  @IsIn([...Object.keys(Currency), ...Object.keys(Fiat)])
+  @IsNotEmpty()
+  public basePair: Currency | Fiat
+
+  @Min(0)
+  @IsOptional()
+  public baseRate?: number
+
+  @IsOptional()
+  @Type(() => CustomerUpdate)
+  @ValidateNested()
+  public customer?: CustomerUpdate
+
+  @Length(3, 3)
+  @IsOptional()
+  @IsIn(Object.keys(Fiat))
+  public accountingCurrency?: Fiat
+
+  @Length(1, 150)
+  @IsNotEmpty()
+  public xpub: string
+
+  @IsNumber()
+  @IsNotEmpty()
+  @Min(1)
+  @Max(2147483647)
```

Comment:
What is this magic number? https://en.wikipedia.org/wiki/2,147,483,647 ? 🙂 Can we make constant somewhere?

Reply:
![image](https://user-images.githubusercontent.com/52315405/148401007-b7e5bb97-18cf-46a8-be7e-e6ebfa5fd279.png)

I got it from the docs, it's the max value for int32
we can but maybe in the other PR, what do you say?:)

Reply:
Ok, I would just put somewhere `const MAX_INT32_VALUE = 2147483647;` and use it. Ok to do it later. 🙂

Reply:
2^32-1